closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -71,6 +73,8 @@ public final class RestrictedSecurity {
 
     private static RestrictedSecurityProperties restricts;
 
+    private static List<String> alteredProperties = new ArrayList<>();
```

Comment:
This should be `final`, a set, and suggest `unmodifiableProperties` is a better name:
```java
    private static final Set<String> unmodifiableProperties = new HashSet<>();
```

Reply:
Changed.